lib/aquilon/worker/templates/cluster.py

```diff
@@ -188,12 +188,7 @@ def get_key(self, exclusive=True):
         return CompileKey.merge(keylist)
 
     def body(self, lines):
-        # This is required to be able to override LOADPATH
-        if self.config.getboolean("panc", "include_pan"):
-            pan_include(lines, ["pan/units", "pan/functions"])
-            lines.append("")
-
-        # Okay, here's the real content
+        pan_include(lines, ["pan/units", "pan/functions"])
```

Comment:
I am really confused to see this happened: hosts.py have the lines.append("") after the pan_include where cluster.py and metacluster.py do not...

Reply:
@urbonegi the easiest is that you tell me what is your expectation for blank lines. My original PR was one after the include of `declarations.pan` and one after the include of pan core templates but you commented, if I understood properly, that it was too much. Is what is currently in `host.py` what you want everywhere?

Reply:
I see now that the host.py was inconsistent in blank lines initially (https://github.com/quattor/aquilon/commit/1ada34baa258498b6e2e81808ae9501cc44e3c5c) - so that is fine. Revert here without changing is fine.